com.bitwarden.desktop.yaml

```diff
@@ -51,8 +51,12 @@ modules:
       - type: script
         dest-filename: bitwarden.sh
         commands:
-          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
-          - exec zypak-wrapper /app/Bitwarden/bitwarden "$@"
+          - export TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
+          - if [ "${XDG_SESSION_TYPE}" == "wayland" ]; then
+          - zypak-wrapper /app/Bitwarden/bitwarden --enable-features=UseOzonePlatform --ozone-platform=wayland "$@"
+          - else
+          - zypak-wrapper /app/Bitwarden/bitwarden "$@"
+          - fi
```

Comment:
Just my two cent: the structure of it is displeasing. I suggest writing a separate bash script.